dashboard/src/app/index.module.ts

```diff
@@ -102,7 +110,11 @@ angular.element(document).ready(() => {
     // load Keycloak
     return keycloakLoad(keycloakSettings).then(() => {
       // init Keycloak
-      return keycloakInit(keycloakAuth.config);
+      var useNonce: boolean;
+      if (typeof keycloakSettings['che.keycloak.use_nonce'] == 'string') {
+        useNonce = keycloakSettings['che.keycloak.use_nonce'].toLowerCase() == 'true';
```

Comment:
shouldn't be === instead of ==

Reply:
fixed in commit https://github.com/eclipse/che/pull/8614/commits/b4604216a28c11a500d754e81df000213827762f